src/control/server/server_utils.go

```diff
@@ -109,11 +109,38 @@ func connectServer(ctlPort int, resolver resolveTCPFn, listener netListenFn) (*n
 	return ctlAddr, lis, nil
 }
 
-func netInit(ctx context.Context, log *logging.LeveledLogger, engineCount int) (context.Context, func(), error) {
+// Provide special handling for the ofi+verbs provide.
+// Mercury uses the interface name such as ib0, while OFI uses the device name
+// such as hfi1_0 CaRT and Mercury will now support the new OFI_DOMAIN
+// environment variable so that we can specify the correct device for each.
+func updateFabricEnvars(ctx context.Context, ec *engine.Config) error {
```

Comment:
nit: The comment indicates that this function is specifically for special handling for ofi+verbs, but the function name implies some generic functionality. I think it's probably a good idea to keep the function name but trim the function comment to something more generic about adjusting the engine fabric configuration and move the details about mercury down into the body above the relevant code. This way if we need to do other similar adjustments we already have a place to do it and we don't need to change these comments.